cnvlib/autobin.py

```diff
@@ -17,8 +17,8 @@ def midsize_file(fnames):
 
     If an even number of files is given, selects the file just below the median.
     """
-    return sorted(fnames, key=lambda f: os.stat(f).st_size
-                 )[len(fnames) // 2 - 1]
+    assert fnames, 'No files provided to calculate the median size.'
```

Comment:
I think this should be a ValueError instead of an assert, since the reported problem would be with the function's input, not with the logic or internal consistency of the function itself.

Thanks for fixing the math bug.